README.md

```diff
@@ -60,22 +60,46 @@ A third-party plugin is required to enable syntax coloring. The Che4z basic stac
 	
 ## Copybook Support
 
-The LSP for COBOL extension can retrieve copybooks used in your projects from the mainframe and download them locally. You can open copybooks in your IDE and make use of the copybook support features of the extension.
+The COBOL Language Support extension supports copybooks called by your source code that are stored in a folder in your workspace. If your copybooks are stored in mainframe data sets, you can use a Zowe CLI z/OSMF profile to automatically download them from the mainframe to your workspace. You specify the local folders and mainframe data sets that contain copybooks used in your project in the workspace settings.
```

Comment:
in a folder -->  in a local folder.  

We are introducing this  term, so we firstly need to mention it here.

Reply:
Done in latest commit